rootfs/standard/usr/share/mynode_apps/datum/scripts/uninstall_datum.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+source /usr/share/mynode/mynode_device_info.sh
+source /usr/share/mynode/mynode_app_versions.sh
+
+echo "==================== UNINSTALLING APP ===================="
+
+rm -rf /opt/mynode/datum/
```

Comment:
There should be no need for lines 8-9. It's done automatically and /mnt/hdd/mynode/datum never seems to be used.